package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "app",
+  "private": "true",
+  "dependencies": {
+    "@hotwired/turbo-rails": "^7.3.0",
+    "axios": "^1.4.0",
+    "react-redux": "^8.1.0",
+    "react-router-dom": "^6.12.1",
```

Comment:
- [ ] It would be better if we add `react-dom` to our dependencies, in the `javascript/application.js` file we used `React-Dom` without having installed it into our project